### PR TITLE
Created Project Info Page

### PR DESCRIPTION
**Pull request by AnimatorJoe**
Styling still needed

### CODE REVIEW

File: client/src/views/Project.vue [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    <div class="main">
+      <div class="row" style="padding-top: 20px">
+        <div class="col text-info1">
+          <h1>{{ this.project.name }}</h1>
+          <p v-if="this.project.looking_for_new_members">
+            is looking for applicants!
+          </p>
+          <div class="row">
+            <p
+              v-for="(f, i) in JSON.parse(this.project.search_filters)"
```
Comment (ShahamNoorani) [inline client/src/views/Project.vue:13]: Lets try not to do this in the html, but either in the mounted or a separate function. Some error handling for values that cannot go through a JSON parse should also be put into place.

File: client/src/views/Project.vue [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            <p v-for="(m, i) in projectMembers" :key="i">
+              {{ m.first_name }} {{ m.last_name }}
+              <br />
+            </p>
+          </div>
+          <br />
+          <div>
+            <p>Links:</p>
+            <a
+              v-for="(link, i) in JSON.parse(this.project.links)"
```
Comment (ShahamNoorani) [inline client/src/views/Project.vue:45]: Same as above.

Comment (ShahamNoorani) [inline client/src/views/Project.vue:45]: same as above

File: client/src/views/Project.vue [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+      var event = await Projects.get(id, accessToken);
+      this.project = event.data;
+
+      JSON.parse(this.project.members).forEach(async (m) => {
+        var response = await Users.get(m, accessToken);
+        this.projectMembers.push(response.data);
+      });
+
+      JSON.parse(this.project.applicants).forEach(async (applicantObject) => {
```
Comment (ShahamNoorani) [inline client/src/views/Project.vue:86]: Very well written!

File: client/src/views/Project.vue (hunk 1 above)
Comment (AnimatorJoe) [reply client/src/views/Project.vue:13]: i see

**Pull request merged**